compose/mpp/demo/src/uikitMain/kotlin/androidx/compose/mpp/demo/bugs/AnimationFreezeBug.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.mpp.demo.bugs
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.Switch
+import androidx.compose.mpp.demo.Screen
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import platform.UIKit.UIApplication
+import platform.Foundation.NSURL
+
+val AnimationFreezeBug = Screen.Example("AnimationFreezeBug") {
```

Comment:
Can we write the automated test case instead of adding another "bug" page in mpp?

Reply:
No, we can't automate this test.